docker-compose.yml

```diff
@@ -11,8 +11,34 @@ services:
     ports:
       - 5432:5432
     volumes:
-      - contact-tracing-pg-data:/var/lib/postgresql/data
-      
+      - covid-green-pg-data:/var/lib/postgresql/data
+
+  covid-green-db-migrations:
+    container_name: covid-green-db-migrations
+    build:
+      context: .
+      dockerfile: Dockerfile-migrate
+    environment:
+      - DB_HOST=covid-green-db
+      - DB_READ_HOST=covid-green-db
+    volumes:
+      - ${PWD}:/covid-green
```

Comment:
Just looking for clarification on what this line does?  This maps the WORKDIR to the `covid-green` volume correct?

Reply:
Yes, that's correct.